chromium_src/third_party/blink/public/common/permissions/permission_utils.h

```diff
@@ -16,16 +27,15 @@
   BRAVE_REFERRERS,                  \
   BRAVE_COOKIES,                    \
   BRAVE_SPEEDREADER,                \
-  BRAVE_ETHEREUM,                   \
-  BRAVE_SOLANA,                     \
+  BRAVE_WALLET_PERMISSION_TYPES     \
   BRAVE_GOOGLE_SIGN_IN,             \
   BRAVE_LOCALHOST_ACCESS,           \
   BRAVE_OPEN_AI_CHAT,               \
-  BRAVE_CARDANO,                    \
   NUM
```

Comment:
Chromium discourages such deletions: 
https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/public/common/permissions/permission_utils.h;l=22-23
Most likely it OK to leave these enums as long as they are never referenced when wallet is cut out

Reply:
is the order important here? Does these map to int values that get persisted?